docs/source/format/Flight.rst

```diff
@@ -90,9 +90,14 @@ A client that wishes to download the data would:
    An endpoint contains a list of locations (server addresses) where
    this data can be retrieved from, and a ``Ticket``, an opaque binary
    token that the server will use to identify the data being
-   requested. There is no ordering defined on endpoints or the data
-   within, so if the dataset is sorted, applications should return
-   data in a single endpoint.
+   requested. If ``FlightInfo.ordered`` is set, returned endpoints are
+   in the same order as the data. Otherwise, there is no ordering
+   defined on endpoints or the data within. The client can read
+   ordered data by reading data from returned endpoints in order from
+   front to back. Note that a client may ignore
+   ``FlightInfo.ordered``. If an ordering is important and the client
+   may ignore ``FlightInfo.ordered``, applications should return data
+   in a single endpoint.
```

Comment:
Here is some suggested updates to this wording based on the mailing list discussion https://lists.apache.org/thread/0q1s84p17rtdz3q81wvylckob1hsx83z

I am not sure this is correct but I tried to encode my understanding

```suggestion
   requested. 
   
   If ``FlightInfo.ordered`` is true, this signals there is some
   order between data from different endpoints. 
   Clients should produce the same results as if the data returned 
   from each of the endpoints was concatenated, in order, from front to back. 
   If ``FlightInfo.ordered`` is not set, the client may return data  from from any 
   of the endpoints in arbitrary order. Data from any specific endpoint
   must be returned in order, but the data from different endpoints may be 
   interleaved to allow parallel fetches.  
   Note that since some clients may ignore ``FlightInfo.ordered``,
   if ordering is important and client support can not be ensured, 
   servers should return a single endpoint.
```

Reply:
Thanks!!!

> data  from from

Is this a typo of `data from`?

@lidavidm Could you also review this suggestion?

Reply:
Thanks for the suggestion - the new wording sounds good to me (Kou already pointed out the typo)

Reply:
Updated wording:

```suggestion
   requested. 
   
   If ``FlightInfo.ordered`` is true, this signals there is some
   order between data from different endpoints. 
   Clients should produce the same results as if the data returned 
   from each of the endpoints was concatenated, in order, from front to back. 
   If ``FlightInfo.ordered`` is not set, the client may return data  from any 
   of the endpoints in arbitrary order. Data from any specific endpoint
   must be returned in order, but the data from different endpoints may be 
   interleaved to allow parallel fetches.  
   Note that since some clients may ignore ``FlightInfo.ordered``,
   if ordering is important and client support can not be ensured, 
   servers should return a single endpoint.
```

Reply:
Thanks! Applied!